apps/src/feedback.js

```diff
@@ -85,7 +85,7 @@ FeedbackUtils.prototype.displayFeedback = function(options, requiredBlocks,
   var displayShowCode = this.studioApp_.enableShowCode && canContinue && !showingSharing;
   var feedback = document.createElement('div');
   var sharingDiv = (canContinue && showingSharing) ? this.createSharingDiv(options) : null;
-  var showCode = displayShowCode ? this.getShowCodeElement_(options) : null;
+  var showCode = displayShowCode && this.studioApp_.enableShowLinesCount ? this.getShowCodeElement_(options) : null;
```

Comment:
Nit, but I would add `this.studioApp_.enableShowLinesCount` check to line 85 instead of here.